Creazione Feature_Elica_intubata_Calcolo_intensità_adimensionale_vort…

MaxOli95 opened the pull request:
Feature_Elica_intubata_Calcolo_intensità_adimensionale_vortice_ad_anello

MaxOli95 committed:
Creazione Feature_Elica_intubata_Calcolo_intensità_adimensionale_vortice_ad_anello

MaxOli95 committed:
Funzione rinominata perchè ecceddeva il numero di caratteri

MaxOli95 committed:
Name fixed

MaxOli95 committed:
Aggiunto la Cartella TestCases

MaxOli95 committed:
Aggiunto una sottocartella

MaxOli95 committed:
Funzione che riproduce la funzione universale graficamente

MaxOli95 committed:
Correzione

GiuseppeMarino commented on Feature_Elica_Intubata_Calcolo_Vortice_ad_Anello.m:
```diff
@@ -1,7 +1,71 @@
+Iy=integral(R2,0,2*pi);
+Iz=integral(R3,0,2*pi);
+%Adimensionalizzazione;
+fx=(r*z*D34/4)*Ix;
```
Ipotesi divisione per 2 piuttosto che per 4 nelle righe54/55/56 per una differente adimensionalizzazione del problema. In questa modo, i valori sono confrontabili con errori minimi rispetto i test cases.

GiuseppeMarino commented on Feature_Elica_Intubata_Calcolo_Vortice_ad_Anello.m:
```diff
@@ -1,7 +1,71 @@
+r=(D14/2)*0.95;
+x=D34/2;
+z=ch*(3/4);
+y=0;
```
y=0 come definito dal prof. Tognaccini

EttoreSaetta commented:
> Riga 42: Riduzione raggio vortice ad anello fattore 0.95. L'ottimo si ottiene con un fattore pari a 0.85.

In base a cosa riducete il raggio del vortice ad anello?

> Riga 45: Coordinata y=0 così come da pdf.
> Riga 54/55/56: Differente adimensionalizzazione delle velocità definite f . Divisione per un fattore 2 piuttosto che un fattore 4.
> 

Da dove esce il 2?

> Questi due accorgimenti permettono una quasi totale corrispondenza tra le curve ricostruite graficamente (funzione Universale2) e l'integrazione matematica descritta dallo script corrente.

EttoreSaetta commented on Feature_Elica_Intubata_Calcolo_Vortice_ad_Anello.m:
```diff
@@ -0,0 +1,71 @@
+%Adimensionalizzazione;
+fx=(r*z*D34/4)*Ix;
+fy=(r*z*D34/4)*Iy;
+fz=(r*D34/4)*Iz;
```
Questi non sono corretti:
 `fx=(r*z*D14/4)*Ix;`
 `fy=-(r*z*D14/4)*Iy;`
 `fz=-(r*D14/4)*Iz;`
Vedete l'equazione 4.18

GiuseppeMarino replied:
Correzione sul codice effettuata.

GiuseppeMarino commented:
Per quanto riguarda l'adimensionalizzazione era solo una ipotesi differente rispetto quella consigliata. Si trattava di un trial and error dato che inizialmente il fattore di differenza fra i due metodi era prossimo a due in molti casi.

MaxOli95 committed:
Correzione adimensionalizzazione con il diametro a 1/4 e correzione variabile di output

MaxOli95 committed:
Cancellazione funzione Universale2 che diagrammava con metodo grafico la funzione f

MaxOli95 committed:
- Cambio nome in uno più sintetico
- aggiunto header
- traduzione in inglese
- commenti aggiuntivi

MaxOli95 committed:
-Feature Rinominata
-Aggiunta Warning

MaxOli95 committed:
-Aggiunta altri commenti e correzioni varie

MaxOli95 committed:
Correzione errori

MaxOli95 committed:
aggiornamento

MaxOli95 committed:
Documentazione(Bozza)

MaxOli95 committed:
Documentation_RVortexInt

MaxOli95 committed:
Correzione distanza punto di controllo dal vortice ad anello

MaxOli95 committed:
-Eliminazione della GUI perchè ritenuta inutile
-Modifica dell'input e conseguente eliminazione della creazione della geometria della carenatura con variazione di diametri lineare
-Conseguente correzione dell'header
-Conseguente modifica dei warnings

MaxOli95 committed:
Delete Documentation_RVortexInt.pdf

Non più attuale

MaxOli95 committed:
Documentazione funzione

Versione definitiva

EttoreSaetta commented on Feature_RVortexInt.m:
```diff
@@ -0,0 +1,86 @@
+function [fx] = RVortexInt(ch,D14,D34)
```
Move this one under the header. The header should be at the top.

EttoreSaetta reviewed (commented):
You are almost ready to merge: just one comment, and move the documentation in the `doc` folder.
Add a test case.

MaxOli95 committed:
Header spostato in alto

MaxOli95 committed:
Add files via upload

Creazione cartella doc per la documentation della funzione RVortexInt (Ring Vortex Intensity)

MaxOli95 committed:
Delete Documentation_RVortexInt.pdf

spostata nella cartella doc

MaxOli95 committed:
Add files via upload

Test Case aggiunto

MaxOli95 commented:
Aggiunta la cartella TestsCases, la cartella doc nella quale si é inserita
la documentazione e spostato l'header in alto

Il mar 26 gen 2021, 09:14 SaettaE <notifications@github.com> ha scritto:

> *@EttoreSaetta* commented on this pull request.
>
> You are almost ready to merge: just one comment, and move the
> documentation in the doc folder.
> Add a test case.
> ------------------------------
>
> In Feature_RVortexInt.m
> <https://github.com/TAARG-Education/Eli-TAARG/pull/18#discussion_r564317861>
> :
>
> > @@ -0,0 +1,86 @@
> +function [fx] = RVortexInt(ch,D14,D34)
>
> Move this one under the header. The header should be at the top.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/TAARG-Education/Eli-TAARG/pull/18#pullrequestreview-576105078>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARUTWGNVQ6NQXLWPEESDTR3S3Z2WTANCNFSM4UVLOICQ>
> .
>

EttoreSaetta commented:
@MaxOli95 
> Aggiunta la cartella TestsCases, la cartella doc nella quale si é inserita la documentazione e spostato l'header in alto

Il test case deve essere in una sottocartella della cartella `TestCases`. Quindi all'interno di `TestCases` create una cartella chiamata `RingVortex` ed all'interno di questa inserite il vostro test case.

MaxOli95 committed:
Delete TestCases directory

Delete

MaxOli95 committed:
Add files via upload

Aggiunta TestCases/RingVortexIntensity/main.RVortex

MaxOli95 commented:
Fatto

Il mar 26 gen 2021, 10:04 SaettaE <notifications@github.com> ha scritto:

> Aggiunta la cartella TestsCases, la cartella doc nella quale si é inserita
> la documentazione e spostato l'header in alto
>
> Il test case deve essere in una sottocartella della cartella TestCases.
> Quindi all'interno di TestCases create una cartella chiamata RingVortex
> ed all'interno di questa inserite il vostro test case.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/TAARG-Education/Eli-TAARG/pull/18#issuecomment-767403647>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARUTWGKZQDSAKWX5L3JOSU3S32AR7ANCNFSM4UVLOICQ>
> .
>

EttoreSaetta reviewed (approved):
LGTM.

MaxOli95 commented:
<!--
/* Font Definitions */
@font-face
	{font-family:"Cambria Math";
	panose-1:2 4 5 3 5 4 6 3 2 4;}
@font-face
	{font-family:Calibri;
	panose-1:2 15 5 2 2 2 4 3 2 4;}
/* Style Definitions */
p.MsoNormal, li.MsoNormal, div.MsoNormal
	{margin:0cm;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
a:link, span.MsoHyperlink
	{mso-style-priority:99;
	color:blue;
	text-decoration:underline;}
.MsoChpDefault
	{mso-style-type:export-only;}
@page WordSection1
	{size:612.0pt 792.0pt;
	margin:70.85pt 2.0cm 2.0cm 2.0cm;}
div.WordSection1
	{page:WordSection1;}
-->Perfetto!  Inviato da Posta per Windows 10 Da: SaettaEInviato: martedì 26 gennaio 2021 10:30A: TAARG-Education/Eli-TAARGCc: Massimiliano; MentionOggetto: Re: [TAARG-Education/Eli-TAARG] Creazione Feature_Elica_intubata_Calcolo_intensità_adimensionale_vort… (#18) @EttoreSaetta approved this pull request.LGTM.—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or unsubscribe. 
